dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -51,6 +51,14 @@ cookie.secure.desc=Specifies that the session cookies include the secure field.
 cookie.http.only=Include HttpOnly field in session cookies
 cookie.http.only.desc=Specifies that session cookies include the HttpOnly field. Browsers that support the HttpOnly field do not enable cookies to be accessed by client-side scripts. Using the HttpOnly field will help prevent cross-site scripting attacks.
 
+# the word SameSite should not be translated
+cookie.same.site=Include SameSite field in session cookies
+cookie.same.site.desc=Specifies which SameSite attribute value should be used for session cookies.
+cookie.same.site.disabled=Do not set a SameSite field on the session cookies
+cookie.same.site.lax=Set the session cookies SameSite field to Lax
```

Comment:
It'd look better with  no "s" at the end of "cookies." That is, use "session cookie SameSite field."

Reply:
This comment applies to the two following strings as well.